src/hardware/ne2000.cpp

```diff
@@ -24,15 +24,12 @@
 
 #include "ne2000.h"
 
-#if C_NE2000
-
```

Comment:
Why wouldn't you want to build the NE2000 emulation in always?

Reply:
I think I explained it in the comment above…

- Some users won't have slirp or will want it enabled, for various reasons (no slirp dependency, faster compilation speed, builds for environments without network access, etc, etc). Almost all our dependencies are opt-out - *and some people criticize us because they want everything opt-out or outright removed - even SDL2*.
- It gives us an easy option to disable the feature on specific OS if we'll end up finding critical issues last-minute before the release. *This happened in the past, several times, e.g. with SDL_net and FluidSynth - both are opt-in.*
- It makes it easier to port emulator to new systems, and you can't anticipate that slirp will be available everywhere. In fact I am rather sure it won't be available e.g. for WebAssembly builds, so sooner or later we'll need to make the feature opt-in anyway.

Reply:
NE2000 emulation is separate to slirp emulation in this patch set. You don't need any libraries to build the NE2000 backend.

Reply:
> NE2000 emulation is separate to slirp emulation in this patch set. You don't need any libraries to build the NE2000 backend.

These are technically true, but @dreamer's suggesting that in practice it makes sense to tie it all together under one configurable bucket, say, "Ethernet support".  Users will either want it whole-sale or they wont.

Reply:
Done.